typescript/sdk/src/ism/CosmosNativeIsmModule.ts

```diff
@@ -85,16 +99,40 @@ export class CosmosNativeIsmModule extends HyperlaneModule<
       'normalized expectedConfig should be an object',
     );
 
+    // Update the config
+    this.args.config = expectedConfig;
+
     // If configs match, no updates needed
     if (deepEquals(actualConfig, expectedConfig)) {
       return [];
     }
 
-    this.args.addresses.deployedIsm = await this.deploy({
-      config: expectedConfig,
-    });
+    // if the ISM is a static ISM we can not update it, instead
+    // it needs to be recreated with the expected config
+    if (STATIC_ISM_TYPES.includes(expectedConfig.type)) {
+      this.args.addresses.deployedIsm = await this.deploy({
+        config: expectedConfig,
+      });
 
-    return [];
+      return [];
+    }
```

Comment:
does the "static ism types" thing still apply to cosmos? asking because if we can update things like aggregation/multisig isms in-place then might be useful to have that functionality

Reply:
Yes it does, only the RoutingIsm can be updated, the other ones have to be newly deployed if something changes there